pkgs/flox-tests/default.nix

```diff
@@ -54,10 +54,12 @@ in
 
         if command -v flox &> /dev/null
         then
-          export TMP_FLOX_CLI=$(command -v flox)
+          TMP_FLOX_CLI="$(command -v flox)"
```

Comment:
@aakropotkin Why is this better than having it in one line?

Reply:
`export` with `"$( ... )"` has undefined behavior when used in combination with `set -e` and/or `set -o pipefail`.

Some shells will fail to export the variable as the empty string without exiting failure. By splitting the `export` to a separate line error catching will work properly.

This is a bit similar to how `: CMD` works.